src/stores/GuildMemberRoleStore.js

```diff
@@ -138,6 +149,19 @@ class GuildMemberRoleStore extends Collection {
    *   .catch(console.error);
    */
   set(roles, reason) {
+    // eslint-disable-next-line eqeqeq
+    if (roles == undefined) return Promise.resolve(this.member);
```

Comment:
you can just do and remove the eslint disable comment
```suggestion
    if (!roles) return Promise.resolve(this.member);
```

Reply:
I don't see any point to making this change other than it simply looking pretty

Reply:
It's called consistency.
You even explicitly disabled eslint rule so it wouldn't complain about using `==` instead of `===`.

Reply:
read upwards, vlad told me to make this change so take it up with him
https://github.com/discordjs/discord.js/pull/3351#discussion_r297781126

Reply:
![well](https://user-images.githubusercontent.com/31581159/60223093-06dea080-9880-11e9-88ff-86cab35448bc.png)

Reply:
> read upwards, vlad told me to make this change so take it up with him

Reply:
>could be as suggested
Since when that means "use this or whole repo will break?

Reply:
Comment mentions "to also handle null instaces"
will that not handle them?

Reply:
Okay, so now what if i say that because I can't be bothered, I don't want to commit this change, does this not already solve the issue, if so why commit more.

Also this consistency you discussed earlier? Yeah there are multiple instances of if (XXX=== undefined) in that GuildMemberRoleStore that i didn't put in there, so there's not much argument for consistency.

Reply:
Nice try to defend `== undefined` with `=== undefined`
But whatever, this is quite pointless from my side as well.

Reply:
it should be `=== undefined`. There is no reason to support someone explicitly doing `set(null)`

Reply:
> it should be `=== undefined`. There is no reason to support someone explicitly doing `set(null)`

There is no such thing as planning too much for too stupid. You should assume every error case and every stupidity known to man.